nixos/modules/programs/sway.nix

```diff
@@ -1,41 +1,36 @@
-{ config
-, pkgs
-, lib
-, ... }:
+{ config, pkgs, lib, ... }:
 
 with lib;
 
 let
   cfg = config.programs.sway;
 
   wrapperOptions = types.submodule {
-    options = {
-      base = mkOption {
-        type = types.bool;
-        default = true;
-        example = "false";
-        description = ''
-          Make use of the base wrapper to execute extra session commands
-          and prepend a dbus-run-session to the sway command.
+    options =
+      let
+        mkWrapperFeature  = default: description: mkOption {
+          type = types.bool;
+          inherit default;
+          example = !default;
+          description = "Whether to make use of the ${description}";
+        };
+      in {
+        base = mkWrapperFeature true ''
+	  base wrapper to execute extra session commands and prepend a
+          dbus-run-session to the sway command.
```

Comment:
Oh dear, seems like I've messed up the indentation here... :o (probably tabs vs. spaces due to the default Vim configuration of my test VM)